acceptance/python/src/test_divisible_asset.py

```diff
@@ -0,0 +1,176 @@
+# # Divisible assets integration testing
+# This test checks if we can successfully divide assets.
+# The script tests various things like:
+#
+# - create a transaction with a divisible asset and issue them to someone
+# - check if the transaction is stored and has the right amount of tokens
+# - spend some tokens
+# - try to spend more tokens than available
+#
+# We run a series of checks for each step, that is retrieving
+# the transaction from the remote system, and also checking the `amount`
+# of a given transaction.
+#
+# This integration test is a rip-off of our
+# [tutorial](https://docs.bigchaindb.com/projects/py-driver/en/latest/usage.html).
+
+# ## Imports
+# We need some utils from the `os` package, we will interact with
+# env variables.
+# We need the `pytest` package to catch the `AmountError` exception properly.
+# And of course, we also need the `AmountError`.
+import os
+import pytest
+from bigchaindb_driver.exceptions import BadRequest
+
+# For this test case we import and use the Python Driver.
+from bigchaindb_driver import BigchainDB
+from bigchaindb_driver.crypto import generate_keypair
+
+
+def test_divisible_assets():
+    # ## Set up a connection to BigchainDB
+    # Check [test_basic.py](./test_basic.html) to get some more details
+    # about the endpoint.
+    bdb = BigchainDB(os.environ.get('BIGCHAINDB_ENDPOINT'))
+
+    # Oh look, it is Alice again and she brought her friend Bob along.
+    alice, bob = generate_keypair(), generate_keypair()
+
+
+    # ## Alice creates a time sharing token
+    # Alice wants to go on vacation, while Bobs bike just broke down.
+    # Alice decides to rent her bike to Bob while she is gone.
+    # So she creates a time sharing token for her bike.
+    # As you can see in the description one token can be used to ride
```

Comment:
I was confused by this sentence at first because tokens are introduced later by `bdb.transactions.prepare`. What do you think about explaining the correspondence there?

Perhaps, like this:
```
# She prepares a `CREATE` transaction and issues 10 tokens. Bob and Alice agree that each token buys one hour of riding the bike.
```